### PR TITLE
Added Manage My Damn Life to section Calendar & Contacts - CalDAV or …

### PR DESCRIPTION
**Pull request by intri-in**
…CardDAV Web-based Clients

Added Manage my Damn Life to section "Calendar & Contacts - CalDAV or CardDAV Web-based Clients"

Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [AgenDAV](https://agendav.github.io/agendav/) - Multilanguage CalDAV web client with a rich AJAX interface and shared calendars support. ([Source Code](https://github.com/agendav/agendav)) `GPL-3.0` `PHP`
 - [Bloben](https://bloben.com) - CalDAV web client. ([Demo](https://demo.bloben.com/api/app/v1/auth/login-demo?username=demo&password=Bg8v16a4q7gvC&redirect=https://demo.bloben.com/calendar?demo=true), [Source Code](https://github.com/nibdo/bloben-app)) `AGPL-3.0` `Docker`
 - [EteSync Web](https://www.etesync.com/faq/#web-client) - EteSync's official Web-based client (i.e., their Web app). ([Demo](https://client.etesync.com/), [Source Code](https://github.com/etesync/etesync-web)) `AGPL-3.0` `Javascript`
-
+- [Manage My Damn Life](https://intri.in/manage-my-damn-life/) - Manage my Damn Life (MMDL) is a self-hosted front end for managing your CalDAV tasks and calendars. . ([Source Code](https://github.com/intri-in/manage-my-damn-life-nextjs)) `GPL-3.0` `Nodejs`
```
Comment (Rabenherz112) [inline README.md:323]:
```suggestion
- [Manage My Damn Life](https://intri.in/manage-my-damn-life/) - Manage my Damn Life (MMDL) is a self-hosted front end for managing your CalDAV tasks and calendars. ([Source Code](https://github.com/intri-in/manage-my-damn-life-nextjs)) `GPL-3.0` `Nodejs/Docker`

```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
Added Docker since you support install via Nodejs and Docker, as well as added back a new line.

**Comment by nodiscc** [comment]
> This project is in alpha state, so be careful if you're working with production data.

I'd rather not add this until it is considered stable and production-ready.

**Comment by Rabenherz112** [comment]
I thought there were no quality requirements for this list? Did I miss something? (Actually confused and interested for future reviews)

**Comment by nodiscc** [comment]
>  thought there were no quality requirements for this list?

There are none, so I do not outright reject it or close the PR, but I still think we should be careful when adding "young" projects to the list - there are precedents for this, search for `alpha` or `beta` in closed issues/PRs, for example. It was never explicitly added to contribution guidelines because it is hard to define correctly ("all projects must be mature and stable enough"  -> define "mature/stable/enough").

There are multiple reasons for my concern:
- There is no point in recommending alpha software when there already are mature and well-working alternatives (it's a calendar web app, of which plenty already exist - if it was something totally new and unique I'd probably be less careful). After all, the purpose of the list is to help users find good/reliable software, not help developers find users.
- Accepting all kinds of software in an alpha/prototype state would mean accepting 100's of low quality projects (not saying this one is) which would make the list even harder to read and overwhelm users with (poor) choices, defeating its purpose - it's supposed to be curated, even if very lightly. The term "awesome" implies that the software is reliable, at the very minimum.
- In my opinion, risks of data loss amount to security risks

That said, credits to the author for clearly stating that this is not ready for production in the README. There are probably other projects in the list with the same level of battle-testing that do not state it at all (which would make a hard "no alpha" rule somehow unfair, because it blocks projects that are honest about their level of advancement).

I'll be happy to read other points of views on this, or uses sharing their experiences with the software, or suggestions of basic quality requirements/guidelines, or maybe @intri-in could tell us what sets this apart from other web calendar/address book software.

**Comment by Rabenherz112** [comment]
Thank you for the clarification, that has been really helpful.

I agree with your perspective on not accepting low-quality projects, although defining what constitutes a "low-quality project" can be subjective. Alpha/beta software, for instance, can still be great and offer awesome features. Thus, determining quality often becomes a case-by-case evaluation, with different people having different criteria. Where we are back to the same issue, how can I as a reviewer see if it is a "low-quality" project? (I totally agree with you on the points made for this PR, this is more a question for general reviews).

Considering your concern about "young" projects, it might be worth considering the introduction of a minimum lifetime requirement. This requirement would filter out newly created projects that are more prone to bugs and might still be in the early development stages. Additionally, it would provide insights into the developers' commitment to maintaining and updating their software. For example, setting a minimum requirement of two months (just an arbitrary number I choose right now) would allow us to observe if the project has received updates after the initial release or if it has been abandoned.

**Comment by intri-in** [comment]
Hi @nodiscc, I understand your apprehensions regarding adding alpha projects to this list, so I'd like to say a few things in support of approving this merge request. 

> There is no point in recommending alpha software when there already are mature and well-working alternatives (it's a calendar web app, of which plenty already exist - if it was something totally new and unique I'd probably be less careful).

MMDL is a **task app and a calendar**, and as far as i know, no open-source self hosted alternatives like this exists. For example, other projects in the section "Calendar & Contacts - CalDAV or CardDAV Web-based Clients" don't fully support tasks. 

- AgenDAV doesn't support tasks
- Bloben supports them but only a few fields, and doesn't support tasks hierarchy. 
- EteSync is more focused on syncing and E2E  rather than being a client for users to manage tasks/calendars. 

Other task apps on this list (such as those listed under section "Task Management & To-do Lists") either don't support both calendars and tasks, or don't support CalDAV

Additionally, Bloben, which is on this list is not production ready as well [Source](https://docs.bloben.com/docs/intro). 

Since MMDL is my own project, I am biased, of course, but I do believe adding it to the list (even if it is a young project) could provide users with a much needed option.

**Comment by Rabenherz112** [comment]
> Might be worth opening an issue specifically for this.

Sure, I created a separate Issue in #3880 , where we and others can discuss this Idea more in-depth.

> In your opinion, is it useful for users of the list, to have non-production-ready software listed?

From my viewpoint, the answer to this question depends on various factors. If I have a specific requirement, such as a manga reader like Komga (https://github.com/gotson/komga), the readiness of the software (whether it's in alpha, beta, or production-ready) may not matter as much to me. As long as it functions properly and doesn't pose any security risks, I would be content using the software because it provides the value I seek (functionality, design, etc.). However, the situation changes rapidly when I need the software for critical purposes, like a firewall such as Opensense (https://opnsense.org/). In such cases, my priority shifts quickly towards stable, consistently operational software, and the specific features become less important.

While I generally don't have an issue with advocating for a list that only includes more mature software, I believe that doing so would exclude a significant amount of excellent software from being featured. If the intention is to enforce such criteria, it should also be clearly stated in the contribution guidelines.

**Review by nodiscc — APPROVED** [review]
Considering arguments that have been made in this PR's comments, and https://github.com/awesome-selfhosted/awesome-selfhosted/issues/3880 having been opened to discuss whether it is appropriate or not to list alpha-quality software, approving.

**Pull request merged**